app/helpers/image_helper.rb

```diff
@@ -0,0 +1,17 @@
+module ImageHelper
+  def image?
+    image.attached?
+  end
+
+  def thumb
+    image.variant(resize: "700x500!")
+  end
+
+  def small_thumb
```

Comment:
Ez kell? A `thumb` is 700x500-as. Ha szukseg van ra akkor legyen kisebb ha nem akkor dobjuk ki szerintem.

Reply:
Nem volt sehol sem használva, kivettem.